### ISSUE TITLE
N-queens problem in Python

### ISSUE THREAD

**Issue by mgncodes** [Algo-Phantoms/Algo-Tree#1320]
> Hello! I'd like to add solutions to some famous backtracking problems in Python 3 as a part of GSSoC '21. It would be very helpful if you assign this issue to me.
> @rudrakshi99
> @raksha009

Heyy, this issue wasn't assigned to you. Plz create a pr only after the issue is assigned. 
Also, if you wanna work on this problem. Plz create a different issue addressing to only 1 problem. Closing your pr for now.

_Originally posted by @raksha009 in https://github.com/Algo-Phantoms/Algo-Tree/issues/42#issuecomment-797394478_

Hi, sorry for the mistake. Now I have made a separate issue. Please assign me this issue. @raksha009

**Comment by Naman-12911** [comment]
Hello sir I want solve this problem with in 2 days please assign me under GSSOC.

**Comment by AparnaJha17** [comment]
@tarun26091999 @shreyanspoddar @OjusWiZard @nakul-19 @anjul2000 @rudrakshi99 @raksha009 @plazzy99
can i do this problem in C++

**Comment by raksha009** [comment]
Issue present at #381

**Comment by AparnaJha17** [comment]
Can I work on this issue?
